portal/models/message.py

```diff
@@ -37,6 +39,18 @@ def send_message(self):
         message.html = fill(self.body, width=280)
         mail.send(message)
 
+        user = User.query.filter_by(email=self.sender).first()
+        user_id = user.id if user else self.user_id
```

Comment:
Hmm - the truth is, it's often the "system" sending the email, not the user.

@mcjustin do we need some sort of system account for this?  Little misleading to suggest the user sent themselves an email reminder, for example.

Reply:
@pbugni good point, yes we should distinguish that this was done by the system automatically.